src/main/java/com/github/vkorobkov/jfixtures/domain/Value.java

```diff
@@ -20,6 +20,14 @@
     @XmlAttribute
     private final ValueType type;
 
+    public static Value of(Object value) {
+        if (value instanceof Value) {
```

Comment:
Can it be simplified?
like
if (value instanceof Value) ? return (Value)value : return new Value(value);